### ISSUE TITLE
The acc of the provided checkpoint is very low

### ISSUE THREAD

**Issue by ghost** [sutd-visual-computing-group/Re-thinking_MI#1]
Hello!
I used the checkpoints of the target classifier VGG16 and IR152 provided by you and tested the classification accuracy.
Why is the acc only 9.97%?

**Comment by ngoc-nguyen-0** [comment]
Hi,

To ensure accurate evaluation, I recommend testing your evaluation code with the pretrained model provided by KEDMI author. You can find the model at the following GitHub repository: https://github.com/SCccc21/Knowledge-Enriched-DMI

If you encounter low classification accuracy, it's possible that there might be some issues in your evaluation code, such as the dataloader. I suggest double-checking your implementation for any bugs or errors.

Ngoc
